test-cypress/steps/import-steps.js

```diff
@@ -0,0 +1,225 @@
+/**
+ * Reusable functions for interacting with the import page.
+ */
+class ImportSteps {
+
+    static visitUserImport(repository) {
+        ImportSteps.visitImport('user', repository);
+    }
+
+    static visitServerImport(repository) {
+        ImportSteps.visitImport('server', repository);
+    }
+
+    static visitImport(type, repository) {
+        cy.visit('/import#' + type);
+
+        if (repository) {
+            cy.selectRepo(repository);
+        }
+
+        cy.get('#import-' + type).should('be.visible');
+
+        return ImportSteps;
+    }
+
+    static openImportURLDialog(importURL) {
+        cy.get('#import-user .import-from-url-btn').click();
+        cy.get('.url-import-form').should('be.visible');
```

Comment:
This check is redundant as cypress ensures element is visible before executing the `type` action

Reply:
Removed it